addons/web_editor/static/src/js/backend/fields.js

```diff
@@ -54,7 +54,7 @@ var FieldTextHtmlSimple = basic_fields.DebouncedField.extend(TranslatableFieldMi
         if (this._getValue() !== this.value) {
             this._isDirty = true;
         }
-        this._super.apply(this, arguments);
+        return $.when.apply($, this.SummernoteManager._saveCroppedImages(this.$content).then(this._super.bind(this)));
```

Comment:
- Since you are calling the method from the outside, it should not be private, make it public in the `SummernoteManager`.
- You don't use `$.when` correctly here, you 'apply' a unique deferred... I guess you want to do this:
```javascript
var self;
var _super = this._super;
var args = arguments;
return saveStuff().then(function () {
    return _super.apply(self, args);
});
```